fields/types/sortableRelateds/SortableRelateds.js

```diff
@@ -113,30 +109,28 @@ module.exports = Field.create({
     });
   },
 
-  loadOptions(postIds) {
+  loadOptions(input, callback) {
     const filters = this.buildFilters();
     xhr({
-      url: Keystone.adminPath + '/api/' + this.props.refList.path + '?basic&' + filters,
+      url: Keystone.adminPath + '/api/' + this.props.refList.path + '?basic&search=' + input + '&' + filters,
       responseType: 'json',
     }, (err, resp, data) => {
       if (err) {
         console.error('Error loading items:', err);
-        return;
+        return callback(null, []);
       }
-      data.results.forEach(post => {
-        if (post) {
-          this._options.push({ label: post.slug, value: post.id });
-          this.cacheItem(post);
-        }
+      data.results.forEach(this.cacheItem);
```

Comment:
Question: Do we still need to check whether items in `data.results` is undefined/null or not ?

Reply:
fixed as above.